vue_example/App.vue

```diff
@@ -1,31 +1,25 @@
 <template>
   <div id="app">
-    <a href="http://zondax.ch">
-      <img
-        alt="Vue logo"
-        src="./assets/logo.png"
-        width="300"
-      >
-    </a>
-    <h1>Cosmos - Ledger integration example</h1>
-    <CosmosLedger msg="Minimal Cosmos+Ledger delegation example" />
+    <img alt="Zondax logo" src="./assets/logo.png" width="300" />
+    <h1>Kusama integration example</h1>
```

Comment:
lol

Reply:
oops :1st_place_medal: :)
That was trying to unify examples/formatting to minimize integration efforts.. !